runtime/compiler/optimizer/InlinerTempForJ9.cpp

```diff
@@ -4012,8 +4012,10 @@ bool TR_MultipleCallTargetInliner::inlineSubCallGraph(TR_CallTarget* calltarget)
     * 1. It's a JSR292 related method. This condition allows inlining method handle thunk chain without inlining the leaf java method.
     * 2. It's force inline target
     */
+   TR::Node *callNode = NULL; // no call node has been generated yet
```

Comment:
Should the comment above this code be updated to reflect the 3rd case that has been added now ?

Reply:
Sure, I'll update the comment